compact.go

```diff
@@ -482,6 +480,14 @@ func (c *LeveledCompactor) write(dest string, meta *BlockMeta, blocks ...BlockRe
 	if err := renameFile(tmp, dir); err != nil {
 		return errors.Wrap(err, "rename block dir")
 	}
+
+	level.Info(c.logger).Log(
+		"msg", "compact blocks",
+		"count", len(blocks),
```

Comment:
Maybe we should also just log all input blocks. A bit noisy but at least we can deduce the exact history in debugging cases.

Reply:
As mentioned in the description - I can deduct that easily given that I know what is the state on startup. But I guess  there is no harm in printing these here too.

Reply:
Yea, seems like a lot of mental work though compared to just having it printed right there.